OperatingSystem-Services/Service-SOC/BasicDocumentation/DockerInstall-Wazuh.md

```diff
@@ -0,0 +1,48 @@
+# How to install Docker For Wazuh Purposes
+
+## DISCLAIMER: Do not follow docker docs, follow provided steps
+
+
+Follow these steps:
+
+## Increase max_map_count on your Docker host:
```

Comment:
For a "step by step" guide. That's fine, but we also want to have some form of learning associated with it. Why do you want to do this? Even if it's a superficial, high-level rationale. Just something ya know?